app/views/machine.njk

```diff
@@ -104,4 +71,22 @@
 {% block js %}
 {#  <script src="/js/vendor/jquery.tablesort.min.js"></script>#}
 {#  <script src="/js/machine.js"></script>#}
+ <script>
+  let backupCalendar = [];
+  {% for week in backupCalendar %}
+    {% for day in week %}
+      backupCalendar.push({
+        bucket: "{{day.bucket}}",
+        today: "{{day.today}}",
+        backup: "{{day.bucket.backup}}",
+        id: "{{day.id}}",
+        date: "{{day.date}}",
+      });
+    {% endfor %}
+  {% endfor %}
+ </script>
+ <script src="/js/machine.js"></script>
+ <script>
+  generateCalendar(backupCalendar);
+ </script>
 {% endblock %}
```

Comment:
You don't have to change the pull request -- but I think it would make a lot more sense to reorganize all of this logic into something that could be pulled from an API call via AJAX, after the page loads. It might make sense to type up an enhancement Issue so that we can track that plan for the future.